vignettes/data_analysis_single_dose_treatment_workflow.Rmd

```diff
@@ -55,7 +55,7 @@ To do this you can take a look at the [quality control vignette](https://jpquast
 
 You can read more about specific functions and how to use them by calling e.g. `?median_normalisation` (for the `median_normalisation()` function). Calling `?` followed by the function name will display the function documentation and give you more detailed information about the function. This can be done for any of the functions included in the package.
 
-This document will give you an overview of data preparation, data analysis, data visualisation and data interpretation functions included in **protti**. It will show you how they can be applied to your data. The examples in this file are run on a [published](https://www.nature.com/articles/s41467-020-18071-x) LiP-MS dataset. In the experiment a HeLa cell lysate was treated with different amounts of the drug Rapamycin and it was be shown that the target of Rapamycin (FKBP12) could be successfully identified. For this vignette we are using a filtered version of the original data set which includes only the 10 μM treatment concentration and the control (untreated). For simplicity only 50 proteins are included in the dataset.
+This document will give you an overview of data preparation, data analysis, data visualisation and data interpretation functions included in **protti**. It will show you how they can be applied to your data. The examples in this file are run on a [published](https://www.nature.com/articles/s41467-020-18071-x) LiP-MS dataset. In the experiment a HeLa cell lysate was treated with different amounts of the drug Rapamycin and it could be shown that the target of Rapamycin (FKBP12) could be successfully identified. For this vignette we are using a filtered version of the original data set which includes only the 10 μM treatment concentration and the control (untreated). For simplicity only 50 proteins are included in the dataset.
```

Comment:
```suggestion
This document will give you an overview of data preparation, data analysis, data visualisation and data interpretation functions included in **protti**. It will show you how they can be applied to your data. The examples in this file are run on a [published](https://www.nature.com/articles/s41467-020-18071-x) LiP-MS dataset. In the experiment a HeLa cell lysate was treated with different amounts of the drug Rapamycin and it was shown that the target of Rapamycin (FKBP12) could be successfully identified. For this vignette we are using a filtered version of the original data set which includes only the 10 μM treatment concentration and the control (untreated). For simplicity only 50 proteins are included in the dataset.
```